python/ray/_private/resource_spec.py

```diff
@@ -69,8 +69,12 @@ def to_resource_dict(self):
 
         memory_units = ray_constants.to_memory_units(
             self.memory, round_up=False)
-        object_store_memory_units = ray_constants.to_memory_units(
-            self.object_store_memory, round_up=False)
+
+        if (self.object_store_memory == 0):
+            object_store_memory_units = 0
+        else:
+            object_store_memory_units = ray_constants.to_memory_units(
```

Comment:
Why don't we handle zero correctly here?

Reply:
inside to_memory_units if value is zero then throw error.
So direct set object_store_memory_units = 0.